modules/administration_manual/pages/configuration/files/encryption/enable-encryption.adoc

```diff
@@ -0,0 +1,51 @@
+= Enable the Encryption App
+:experimental:
+:page-partial:
+
+Before you can use encryption you must enable the encryption app. 
+You can do this either from the command-line or from the Web-UI.
+
+== Enable Encryption From the Command-Line
+// tag::enable-encryption-app-via-command-line[]
+To enable the encryption app, run the following command:
+
+[source,console]
+----
+sudo -u www-data php occ app:enable encryption
+----
+
+If the encryption app successfully enables, then you should see the
+following confirmation:
+
+....
+encryption enabled
+....
+
+[NOTE] 
+====
+This should never happen, but the encryption app may not be packaged with your ownCloud installation. 
+If so, you will see the following output when you attempt to enable it:
+
+....
+encryption not found
+....
+
+If that happens, then you need to install it manually. 
+You can do this by cloning the encryption app, using the following command:
+
+[source,console]
+----
+git clone https://github.com/owncloud/encryption apps/encryption
```

Comment:
If I go to the encryption repro, the clone link  provided looks like: ``https://github.com/owncloud/encryption.git``

Reply:
Thanks for spotting that. My mistake.